plugins/slicer/MONAILabel/MONAILabel.py

```diff
@@ -1002,6 +1008,9 @@ def onSaveLabel(self):
 
         logging.info("Time consumed by save label: {0:3.1f}".format(time.time() - start))
 
+    def onClickContour(self):
+        self.editUtil.toggleLabelOutline()
```

Comment:
@lassoan I'd like to toggle the contour/label outline on the segmentation nodes in the scene. Somehow it is not toggling the contours. Am I missing something? I'm trying to follow the code available here: https://github.com/Slicer/Slicer/blob/master/Modules/Scripted/EditorLib/EditUtil.py#L268

Reply:
EditorLib is for the old Editor module, for labelmaps, not for segmentations. 

You can change segmentation visualization options in the display node (vtkMRMLSegmentationDisplayNode) of the segmentation node that you can get by calling segmentationNode.GetDisplayNode().

Reply:
Excellent! Thanks @lassoan !